### PR TITLE
Updated description

### PR DESCRIPTION
**Pull request by WaltWe**
Changed description in README.md to be more accurate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # 2018-Java
-Repository for the 2018 build. Java was used this year. 
+Repository for FRC team 4418's 2018 software build. Java was used this year.
```
Comment (WaltWe) [inline README.md:2]: Description change verified.

**Review by WaltWe — COMMENTED** [review]
Changes look good.

File: README.md (hunk 1 above)
Comment (WaltWe) [reply README.md:2]: However, someone other than me (Will) has to review this change and approve it with the way I set up the repository.

Comment (sploders101) [reply README.md:2]: That's cool!

**Comment by ahegemann** [comment]
Include the comment description and it should be "ready" to go!

**Comment by WaltWe** [comment]
Added the description.

**Review by emhep — APPROVED** [review]

**Pull request merged**